drake/examples/Quadrotor/QuadrotorOutput.h

```diff
@@ -79,11 +90,15 @@ bool encode(const double& t, const QuadrotorOutput<double> & x, drake::lcmt_quad
   Eigen::Map<Eigen::Matrix<double, 6, 1>> lcm_twist(msg.twist);
   Eigen::Map<Eigen::Vector3d> lcm_accelerometer(msg.accelerometer);
   Eigen::Map<Eigen::Vector3d> lcm_gyroscope(msg.gyroscope);
+  Eigen::Map<Eigen::Vector3d> lcm_magnetometer(msg.magnetometer);
+  Eigen::Map<Eigen::Matrix<double, x.num_lidar_points, 1>> lcm_lidar(msg.lidar_returns);
```

Comment:
Nice John!! Lidar fix looks great!

On Wednesday, March 16, 2016, Russ Tedrake <notifications@github.com> wrote:

> In drake/examples/Quadrotor/QuadrotorOutput.h
> <https://github.com/RobotLocomotion/drake/pull/1878#discussion_r56446991>:
>
> > @@ -79,11 +90,15 @@ bool encode(const double& t, const QuadrotorOutput<double> & x, drake::lcmt_quad
> >    Eigen::Map<Eigen::Matrix<double, 6, 1>> lcm_twist(msg.twist);
> >    Eigen::Map<Eigen::Vector3d> lcm_accelerometer(msg.accelerometer);
> >    Eigen::Map<Eigen::Vector3d> lcm_gyroscope(msg.gyroscope);
> > +  Eigen::Map<Eigen::Vector3d> lcm_magnetometer(msg.magnetometer);
> > +  Eigen::Map<Eigen::Matrix<double, x.num_lidar_points, 1>> lcm_lidar(msg.lidar_returns);
>
> travis complained (legitimately). this should be
> QuadrotorOutput::num_lidar_points.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1878/files/2d6440508f1f0ce061f49a9d6eb7efd4a1dc2228#r56446991>
>